README.md

```diff
@@ -19,28 +20,16 @@ docker build -t btcz/bitcoinz-rest-api .
 ```
 
 ## Usage
-To simplying get the container up and running using the following command:
+To get the container up and running quickly, use the following command:
 
 ```
-docker run -it --name bitcoinz-rest-api \
-  -p 3001:3001 \
+docker run -it --name bitcoinz-insight-patched \
   -p 1989:1989 \
-  btcz/bitcoinz-rest-api
-```
+  btcz/bitcoinz-insight-patched
 
-NOTE: The BitcoinZ data lives in `/bitcoinz/data`. It is suggested to mount a shared volume so that
-if the docker container is ever updated or deleted, the node data is still available. You can
-mount a shared volume like so:
-
-
-```
-sudo mkdir -p /opt/storage/bitcoinz/data/
-sudo chmod a+rwx /opt/storage/bitcoinz/data/
-cp bitcoinz.conf /opt/storage/bitcoinz/data/zcash.conf
 docker run -it --name bitcoinz-rest-api \
+  --link bitcoinz-insight-patched
   -p 3001:3001 \
-  -p 1989:1989 \
-  -v /opt/storage/bitcoinz/data/:/bitcoinz/data \
   btcz/bitcoinz-rest-api
```

Comment:
Is this right? I'm getting:
```
Pulling insight (btcz/bitcoinz-rest-api:latest)...                             
ERROR: pull access denied for btcz/bitcoinz-rest-api, repository does not exist or may require 'docker login'                                                 
```

Reply:
No, that's not right. You need to build it first I guess.

Reply:
I'm using docker-compose so this isn't an issue for me but I think it'll cause confusion. Do you want me to push up a fix or do you want to?

Reply:
Let me know if you need me to push up the update to the README.

Reply:
Do it!